resources/translations/cs_CZ/cooling/cool_min_layer_time.md

```diff
@@ -10,4 +10,4 @@ Toto nastavení má tři efekty:
 * Pokud se vrstva tiskne rychleji, než je uvedeno v parametru [Práh 
  normální/maximální rychlosti ventilátoru](cool_min_layer_time_fan_speed_max.md), zvýší se rychlost ventilátoru tak, aby dosáhl parametru [Maximální rychlost ventilátoru](cool_fan_speed_max.md). Pokud je vrstva tak malá, že její tisk trvá kratší dobu, použije se maximální rychlost ventilátoru. Rychlost ventilátoru je mezi nimi interpolována.
 * Pokud vytisknutí vrstvy zabere méně času, než je minimální doba vrstvy, rychlost tisku se zpomalí, takže vždy zabere minimální čas vrstvy.
-* Pokud je tisková hlava příliš pomalá (tisk pomalejší než nastavení parametru [Minimální rychlost](cool_min_speed.md)), bude tisková hlava čekat na konci vrstvy a může ji trochu zvednout.
+* Pokud je tisková hlava příliš pomalá (tisk pomalejší než nastavení parametru [Minimální rychlost](cool_min_speed.md), bude tisková hlava čekat na konci vrstvy a může ji trochu zvednout.
```

Comment:
Oh this one was correct I think. One closing bracket is from the link, the other one from the actual bracketed text that started at the word `tisk`.

Reply:
fixed